components/project-details/caseStudiesPreview/Preview.tsx

```diff
@@ -1,23 +1,23 @@
 import WrapperProjectDetails from '@/HOC/WrapperProjectDetails';
-import { OtherCaseStudiesType } from '@/types';
-import ProjectDetailsHeading from '../reusable/ProjectDetailsHeading';
-import OtherCaseStudyCard from './OtherCaseStudyCard';
+import { PreviewType } from '@/types';
+import SectionHeading from '../../reusable/SectionHeading';
+import StudyCard from './PreviewCard';
 
-const OtherCaseStudies = ({ otherCaseStudies }: OtherCaseStudiesType) => {
+const Preview = ({ otherCaseStudies }: PreviewType) => {
```

Comment:
No need for brackets. I'll stop mentioning it, again it's a style choice but you've used both styles.